Misc/NEWS.d/next/Library/2023-03-16-08-17-29.gh-issue-102748.WNACpI.rst

```diff
@@ -0,0 +1 @@
+:func:`asyncio.iscoroutine` now returns :const:`False` for generators as :mod:`asyncio` does not supports legacy generator based coroutines. Patch by Kumar Aditya.
```

Comment:
(Should probably also wrap this to 80 chars per line)

```suggestion
:func:`asyncio.iscoroutine` now returns :const:`False` for generators as :mod:`asyncio` does not supports legacy generator-based coroutines. Patch by Kumar Aditya.
```

Reply:
Thanks committed manually